react/.eslintrc

```diff
@@ -1,3 +1,11 @@
 {
-  "extends": "vtex-react"
+  "extends": "vtex-react",
+  "parserOptions": {
+    "project": "./react/tsconfig.json"
```

Comment:
🤔  what does it do?

Reply:
it's necessary for some eslint rules, but I don't know why it didn't pickup this definition from [eslint-config-vtex](https://github.com/vtex/typescript/blob/master/packages/eslint-config-vtex/rules/typescript.js#L21-L29), so I had to manually add it.

Reply:
nevermind, all I needed to do was run `yarn upgrade` 😄